packages/addons/service/fd628/sources/addon.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.fd628"
+		name="FD628 Display"
+		version="@PKG_VERSION@"
+		provider-name="LibreELEC">
+    <requires>
+		<import addon="xbmc.python" version="2.1.0"/>
+    </requires>
+    <extension point="xbmc.service"
+		library="default.py"
+		start="startup">
+    </extension>
+    <extension point="xbmc.addon.metadata">
+		<summary>A service for displaying icons on devices with the FD628 VFD display</summary>
+		<description>This service is for displaying the additional icons on devices like the Tanix TX3 mini with the FD628 VFD display</description>
```

Comment:
"Service for controlling FD628 VFD display icons, e.g. Ethernet/WiFi connection status and Time"